tests/subsys/spm/thread_swap/testcase.yaml

```diff
@@ -0,0 +1,4 @@
+tests:
+  spm.secure_services.thread_swap:
+    platform_allow: nrf9160dk_nrf9160ns nrf5340pdk_nrf5340_cpuappns
```

Comment:
```suggestion
    platform_allow: nrf9160dk_nrf9160ns nrf5340pdk_nrf5340_cpuappns nrf5340dk_nrf5340_cpuappns
```

Reply:
Not really sure if this is needed, just check what is in the other tests.

Reply:
Looks like nrf5340dk_nrf5340_cpuappns is not used in any testcase.yaml or sample.yaml files yet.